labelbox/orm/db_object.py

```diff
@@ -107,6 +111,7 @@ def __init__(self, source, relationship):
         self.supports_filtering = True
         self.supports_sorting = True
         self.filter_on_id = True
+        self.value = value
```

Comment:
Can you explain this change / this new instance var?

Reply:
I added a comment on the relationship manager 
``` 
Whether or not to precompute the relationship.
           Useful for objects that aren't directly queryable from the api (relationship query builder won't work)
          Also useful for expensive ToOne relationships 
```
In this case we get `member`s back from a custom query. Since a `member` isn't directly queryable it can't be represented as a dbobject even though we have all of the information to construct the object. This enables us to pass the information to create this object without an additional query.

This is also useful if you have an object that has a relationship that needs to be used for every result. E.g. labeling parameter overrides usually need data rows to do anything useful. However, you need query for each individual data row which. This enables us to do the join on the backend and drastically cut the number of requests.